### ISSUE TITLE
For which of the (GNU) Linux operating systems is the current download intended?

### ISSUE THREAD

**Issue by RonCam** [teejee2008/timeshift#477]
I have recently started using a distro (BunsenLabs Linux), based on Debian, and the docs on their forum say we should only install packages built for the correct version of Debian and not, for example, some package built for Ubuntu.

In the timeshift README, I've searched for Debian-based distro names, but the only information I can get from it, is that Timeshift is "built for 'Linux'".  Does the word 'Linux', used by itself, mean Timeshift is distro-agnostic and version-agnostic, _so long at that distro will accept _.deb_ packages?_

There are some other packages, that might be named __foo-all_.deb,_  where the _'all.deb'_  part of the package name seems to indicate that the package could be installed in any system running the Linux kernel, without regard to its name or version, so long at it accepts '.deb packages'.  In the absence of other information in the README, does timeshift fall into this category?

In other words, is it recommended -- to avoid damaging one's system -- that Timeshift be built from source, on the distro and version, on which it's to be installed?  Or, is this not necessary?

**Comment by RonCam** [comment]
Many thanks for the quick and very complete reply.  
I suspected that might be the case, but no one over on the Bunsen Labs forum was able to give an authoritative reply.  Now I may sleep peacefully, knowing my new BL Linux install isn't on the road to becoming a FrankenDebian!
I'll be sure to link this to the forum thread at Bunsen Labs, where the question was originally posed.
